libraries/chain/database.cpp

```diff
@@ -2190,15 +2192,25 @@ void database::process_funds()
 
       witness_reward /= wso.witness_pay_normalization_factor;
 
+      auto new_sbd = asset( 0, SBD_SYMBOL );
+
+      if( sps_fund.value )
+      {
+         new_sbd = asset( sps_fund, STEEM_SYMBOL ) * feed.current_median_history;
+         adjust_balance( STEEM_TREASURY_ACCOUNT, new_sbd );
+      }
+
       new_steem = content_reward + vesting_reward + witness_reward;
 
       modify( props, [&]( dynamic_global_property_object& p )
       {
          p.total_vesting_fund_steem += asset( vesting_reward, STEEM_SYMBOL );
          if( !has_hardfork( STEEM_HARDFORK_0_17__774 ) )
             p.total_reward_fund_steem  += asset( content_reward, STEEM_SYMBOL );
-         p.current_supply           += asset( new_steem, STEEM_SYMBOL );
-         p.virtual_supply           += asset( new_steem, STEEM_SYMBOL );
+         p.current_supply      += asset( new_steem, STEEM_SYMBOL );
+         p.current_sbd_supply  += new_sbd;
+         p.virtual_supply      += asset( new_steem, STEEM_SYMBOL );
```

Comment:
The virtual supply should be updated with the sps_fund value as well `asset( new_steem + sps_fund, STEEM_VALUE)`. It will be automatically updated later in the block, but we should be proactive here to prevent any inaccuracies that might have unintended side effects in other parts of the code.